openff/interchange/components/toolkit.py

```diff
@@ -93,3 +97,32 @@ def _combine_topologies(topology1: Topology, topology2: Topology) -> Topology:
         topology1_.add_molecule(molecule)
 
     return topology1_
+
+
+def _check_electrostatics_handlers(force_field: "ForceField") -> bool:
+    """
+    Return whether or not this ForceField should have an Electrostatics tag.
+    """
+    # Manually-curated list of names of ParameterHandler classes that are expected
+    # to assign/modify partial charges
+    partial_charge_handlers = [
+        "LibraryCharges",
+        "ToolkitAM1BCC",
+        "ChargeIncrementModel",
+        "VirtualSites",
+        "GBSA",
+    ]
+
+    # A more robust solution would probably take place late in the parameterization
+    # process, but this solution should cover a vast majority of cases with minimal
+    # complexity. Most notably this will *not* behave well with
+    #   * parameter handler plugins
+    #   * future additions to -built-in handlers
+    #   * handlers that _could_ assign partial charges but happen to not assign
+    #       any for some particular topology
```

Comment:
These are fantastic details to capture. Maybe this could be a design note for interchange plugins - Like, one potentialhandler can specify that it requires the existence of another (eg ToolkitAM1BCC requires Electrostatics?)